app/broadcast_message/rest.py

```diff
@@ -37,6 +37,41 @@ def _parse_nullable_datetime(dt):
     return dt
 
 
+def _update_broadcast_message(broadcast_message, new_status, updating_user):
```

Comment:
Does this function also need to check if the updating user has the right permissions for the service?

Reply:
yeah, probably - but i'll leave that until we understand a bit more about what kind of roles we have in a broadcast team. Not quite sure which permissions get to do what bit (eg: which permission do you need to propose a message? which do you need to click approve? will these teams just give every permission to every one, or will they really carefully control who gets what?

(current permissions:)

* manage_users
* manage_templates
* manage_settings
* send_texts
* send_emails
* send_letters
* manage_api_keys
* platform_admin
* view_activity